src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -222,15 +282,18 @@ public int hashCode() {
         return Objects.hashCode(this.format, this.children, this.clickAction, this.hoverAction, this.shiftClickAction);
     }
 
-    @Override
-    public String toString() {
+    protected Objects.ToStringHelper toStringHelper() {
         return Objects.toStringHelper(Text.class)
                 .add("format", this.format)
                 .add("children", this.children)
                 .add("clickAction", this.clickAction)
                 .add("hoverAction", this.hoverAction)
-                .add("shiftClickAction", this.shiftClickAction)
-                .toString();
+                .add("shiftClickAction", this.shiftClickAction);
```

Comment:
Please change the method to return a reduced output, whenever possible.

````java
return Objects.toStringHelper(Text.class)
	.omitNullValues()
	.add("format", this.format.isEmpty() ? null : this.format) // <- Optional, but recommended
	.add("children", this.children.isEmpty() ? null : this.children) // <- Optional, but recommended
	.add("clickAction", this.clickAction.orElse(null))
	.add("hoverAction", this.hoverAction.orElse(null))
	.add("shiftClickAction", this.shiftClickAction.orElse(null));
````
Here an example: `Text.of()` aka empty `Text`
Currently:
`Text{format=TextFormat{style=NoneTextStyle{}, color=NONE}, children=[], clickAction=Optional.empty, hoverAction=Optional.empty, shiftClickAction=Optional.empty, }`

Improved:
`Text{}`